backend/compact-connect/tests/unit/test_app.py

```diff
@@ -95,9 +95,28 @@ def test_synth_sandbox_no_domain(self):
         self._check_no_annotations(app.sandbox_stage.persistent_stack)
         self._check_no_annotations(app.sandbox_stage.ui_stack)
         self._check_no_annotations(app.sandbox_stage.api_stack)
+        self._check_no_annotations(app.sandbox_stage.ingest_stack)
```

Comment:
I've seen this before enough times that I gotta ask...what does this do?

Reply:
We are using the [cdk-nag](https://github.com/cdklabs/cdk-nag) library to sanity-check our infrastructure for security best practices. When `cdk-nag` finds things like a resource without encryption at rest configured, or other general security rules, it adds a finding to the `annotations` for the stack on CDK synthesis. The `_check_no_annotations()` method is looking at `annotations` and asserting that there are no un-suppressed findings outstanding for that stack.

Reply:
awesome, love that. thank you!